targets.toml

```diff
@@ -0,0 +1,460 @@
+os = "ubuntu-latest"
+cpp = true
+dylib = true
+std = true
+run = true
+runners = "qemu-user qemu-system"
+
+[[target]]
+target = "riscv64gc-unknown-linux-gnu"
+os = "ubuntu-latest"
+cpp = true
+std = true
+run = true
+runners = "qemu-user qemu-system"
+
+[[target]]
+target = "s390x-unknown-linux-gnu"
+os = "ubuntu-latest"
+cpp = true
+dylib = true
+std = true
+run = true
+runners = "qemu-user qemu-system"
+
+[[target]]
+target = "sparc64-unknown-linux-gnu"
+os = "ubuntu-latest"
+cpp = true
+dylib = true
+std = true
+run = true
+runners = "qemu-user qemu-system"
+
+[[target]]
+target = "aarch64-unknown-linux-musl"
+os = "ubuntu-latest"
+cpp = true
+dylib = true
+std = true
+run = true
+
+[[target]]
+target = "arm-unknown-linux-musleabihf"
+os = "ubuntu-latest"
+cpp = true
+dylib = true
+std = true
+run = true
+
+[[target]]
+target = "arm-unknown-linux-musleabi"
+os = "ubuntu-latest"
+cpp = true
+dylib = true
+std = true
+run = true
+
+[[target]]
+target = "armv5te-unknown-linux-gnueabi"
+os = "ubuntu-latest"
+cpp = true
+dylib = true
+std = true
+run = true
+
+[[target]]
+target = "armv5te-unknown-linux-musleabi"
+os = "ubuntu-latest"
+cpp = true
+dylib = true
+std = true
+run = true
+
+[[target]]
+target = "armv7-unknown-linux-musleabi"
+os = "ubuntu-latest"
+cpp = true
+dylib = true
+std = true
+run = true
+
+[[target]]
+target = "armv7-unknown-linux-musleabihf"
+os = "ubuntu-latest"
+cpp = true
+dylib = true
+std = true
+run = true
+
+[[target]]
+target = "i586-unknown-linux-musl"
+os = "ubuntu-latest"
+cpp = true
+dylib = true
+std = true
+run = true
+runners = "qemu-user"
+
+[[target]]
+target = "i686-unknown-linux-musl"
+os = "ubuntu-latest"
+cpp = true
+dylib = true
+std = true
+run = true
+runners = "qemu-user"
+
+[[target]]
+target = "mips-unknown-linux-musl"
+os = "ubuntu-latest"
+cpp = true
+dylib = true
+std = true
+run = true
+
+[[target]]
+target = "mipsel-unknown-linux-musl"
+os = "ubuntu-latest"
+cpp = true
+dylib = true
+std = true
+run = true
+
+[[target]]
+target = "aarch64-linux-android"
+os = "ubuntu-latest"
+cpp = true
+std = true
+run = true
+runners = "qemu-user"
+
+[[target]]
+target = "arm-linux-androideabi"
+os = "ubuntu-latest"
+cpp = true
+std = true
+run = true
+runners = "qemu-user"
+
+[[target]]
+target = "armv7-linux-androideabi"
+os = "ubuntu-latest"
+cpp = true
+std = true
+run = true
+runners = "qemu-user"
+
+[[target]]
+target = "thumbv7neon-linux-androideabi"
+os = "ubuntu-latest"
+cpp = true
+std = true
+run = true
+runners = "qemu-user"
+
+[[target]]
+target = "i686-linux-android"
+os = "ubuntu-latest"
+cpp = true
+std = true
+run = true
+runners = "qemu-user"
+
+[[target]]
+target = "x86_64-linux-android"
+os = "ubuntu-latest"
+cpp = true
+std = true
+run = true
+runners = "qemu-user"
+
+[[target]]
+target = "x86_64-pc-windows-gnu"
+os = "ubuntu-latest"
+cpp = true
+std = true
+run = true
+
+[[target]]
+target = "i686-pc-windows-gnu"
+os = "ubuntu-latest"
+cpp = true
+std = true
+run = true
+
```

Comment:
We're missing the commented out `asmjs-unknown-emscripten`, which we should document why it's not enabled.

```yaml
# Disabled for now, see https://github.com/rust-lang/rust/issues/98216
#-{ target: asmjs-unknown-emscripten,         os: ubuntu-latest,  cpp: 1,           std: 1, run: 1 }
```